src/Microsoft.Dnx.Runtime/Project.cs

```diff
@@ -233,12 +236,25 @@ internal static Project GetProjectFromStream(Stream stream, string projectName,
 
             project.Dependencies = new List<LibraryDependency>();
 
+            // Files to be packed along with the project
+            var packInclude = rawProject.ValueAsJsonObject(PackIncludePropertyName);
```

Comment:
So a separate project object? That's not a bad idea either. Also, we should extract ProjectReader out into a package at some point (possibly just a shared source package). We'd still embed it in the runtime, but as @rynowak said it's useful in other situations.